src/Missings.jl

```diff
@@ -165,4 +165,30 @@ function levels(x)
     levs
 end
 
+struct PassMissing{F} <: Function end
+
+@generated (::PassMissing{F})(xs...;kw...) where {F} =
```

Comment:
I don't think the `@generated` is doing anything here except ensuring a new function gets compiled w/ every call.